op-chain-ops/genesis/config.go

```diff
@@ -734,6 +734,19 @@ func NewL2StorageConfig(config *DeployConfig, block *types.Block) (state.Storage
 		"_initialized":  2,
 		"_initializing": false,
 	}
+	storage["GoldToken"] = state.StorageValues{
+		"initialized":  true,
```

Comment:
Cool that they map the variable names here, so that we don't have to use the memory addresses!

Reply:
Agreed ! It took some digging to figure out whether and how they support mappings :-)